nextcloudappstore/settings/base.py

```diff
@@ -198,3 +198,5 @@
 
 LOGIN_REDIRECT_URL = 'home'
 LOGIN_URL = 'account_login'
+
+PLATFORM_VERSIONS = []
```

Comment:
I meant to ask about that. Currently it only works for full semver numbers, like 9.0.0, but we might want to support shorter/broader platform numbers.

I resorted to adding some version numbers to this in `settings/development.py`. Do I put the default in `settings/production.py`?